dali/python/nvidia/dali/plugin/jax/iterator.py

```diff
@@ -172,7 +172,7 @@ def _next_impl(self):
         for category_id, category_name in enumerate(self.output_map):
             category_outputs = self._gather_outputs_for_category(pipelines_outputs, category_id)
 
-            if self._num_gpus == 1:
+            if self._num_gpus == 1 and self._sharding is None:
```

Comment:
`self._num_gpus` is equal to the number of pipelines run by this instance of the iterator. So if we need to distinguish multi node training with one GPU per node (process) from just one GPU training.